config/database.yml.example

```diff
@@ -1,18 +0,0 @@
-default: &default
```

Comment:
One of the common technics is not to store `database.yml` and `secrets.yml` in the repo as they then may have sensitive production data.
In such cases when you clone a rails project, it often has `database.yml.example` and `secrets.yml.example`, so you just need to make your own copies of those files without `.example`.